lib/fs/storage.go

```diff
@@ -227,7 +227,12 @@ func (s Storage) Stat(path string) (storage.FileInfo, error) {
 		return nil, err
 	}
 
-	return info, err
+	inf := &FileInfo{
+		size:         info.Size(),
+		lastModified: info.ModTime(),
```

Comment:
It looks like FileInfo struct contains more fields, I think it makes sence to fill them up as well, function for that are in place

Reply:
@protsack-stephan do you think we need to add all fields?